lib/site_prism/element_container.rb

```diff
@@ -185,32 +181,6 @@ def create_nonexistence_checker(element_name, *find_args)
         end
       end
 
-      def create_waiter(element_name, *find_args)
```

Comment:
Should we deprecate it first?

Reply:
I’m in a couple of minds. But I’d prefer not to. 

What we can do though is deprecate it in the 2.x version. 

Reason why is I only want to do one set of big changes for v3. Not doing another major bump for a while. 

Thoughts @tgaff ??

In theory this release won’t be ready for a while so cutting another release that deprecates this seems valid?

Reply:
We could do a final v2.999 build as an upgrade step with deprecations.